src/platform/test/functional/page_objects/home_page.ts

```diff
@@ -92,11 +99,6 @@ export class HomePageObject extends FtrService {
   async addSampleDataSet(id: string) {
     await this.openSampleDataAccordion();
     await this.retry.waitFor('sample data to be installed', async () => {
-      // count for the edge case where some how installation completes just before the retry occurs
```

Comment:
This handler was removed because the reason for this edge case must have been that the installation completed before retry, because the `testSubjects.find` had a 10 second timeout.